### PR TITLE
Ingest csv review - Global Gliders

### PR DESCRIPTION
**Pull request by cdobs**
Completed review of Global Glider telemetered and recovered ingestion csvs

### CODE REVIEW

File: GA05MOAS-GL364/GA05MOAS-GL364_R00002_ingest.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
-#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,,GA05MOAS-GL364-02-DOSTAM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,,GA05MOAS-GL364-04-CTDGVM000,recovered_host,Not Expected,Glider lost
\ No newline at end of file
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL364/R00002/merged/ga_*.mrg,GA05MOAS-GL364-00-ENG000000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL364/R00002/merged/ga_*.mrg,GA05MOAS-GL364-01-FLORDM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL364/R00002/merged/ga_*.mrg,GA05MOAS-GL364-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL364/R00002/merged/ga_*.mrg,GA05MOAS-GL364-04-CTDGVM000,recovered_host,Not Expected,Glider lost
```
Comment (snwhite) [inline GA05MOAS-GL364/GA05MOAS-GL364_R00002_ingest.csv:5]: Looks good.

File: GA05MOAS-GL470/GA05MOAS-GL470_R00001_ingest.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,,GA05MOAS-GL470-02-DOSTAM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,,GA05MOAS-GL470-04-CTDGVM000,recovered_host,Not Expected,Glider lost
\ No newline at end of file
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL470/R00001/merged/ga_*.mrg,GA05MOAS-GL470-00-ENG000000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL470/R00001/merged/ga_*.mrg,GA05MOAS-GL470-01-FLORDM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL470/R00001/merged/ga_*.mrg,GA05MOAS-GL470-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-GL470/R00001/merged/ga_*.mrg,GA05MOAS-GL470-04-CTDGVM000,recovered_host,Not Expected,Glider lost
```
Comment (snwhite) [inline GA05MOAS-GL470/GA05MOAS-GL470_R00001_ingest.csv:5]: Looks good.

File: GA05MOAS-PG578/GA05MOAS-PG578_R00001_ingest.csv [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-03-FLORTM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-04-FLORTO000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-05-NUTNRM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/GA05MOAS-PG578/R00001/merged-from-glider/ga_*.mrg,GA05MOAS-PG578-06-PARADM000,recovered_host,Not Expected,Glider lost
```
Comment (snwhite) [inline GA05MOAS-PG578/GA05MOAS-PG578_R00001_ingest.csv:8]: Looks good... 
Does NUTNR really not need .moas.gl. in the parser name?

File: GI05MOAS-GL365/GI05MOAS-GL365_D00003_ingest.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
\ No newline at end of file
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL365-00-ENG000000,telemetered,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL365-01-FLORDM000,telemetered,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL365-02-DOSTAM000,telemetered,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL365-04-CTDGVM000,telemetered,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
```
Comment (snwhite) [inline GI05MOAS-GL365/GI05MOAS-GL365_D00003_ingest.csv:5]: Looks good.

File: GI05MOAS-GL365/GI05MOAS-GL365_R00003_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00003/merged/gi*.mrg,GI05MOAS-GL365-00-ENG000000,recovered_host,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00003/merged/gi*.mrg,GI05MOAS-GL365-01-FLORDM000,recovered_host,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00003/merged/gi*.mrg,GI05MOAS-GL365-02-DOSTAM000,recovered_host,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00003/merged/gi*.mrg,GI05MOAS-GL365-04-CTDGVM000,recovered_host,Not Expected,Glider recovered after 6 hours due to GPS issue; no data expected
```
Comment (snwhite) [inline GI05MOAS-GL365/GI05MOAS-GL365_R00003_ingest.csv:5]: Looks good.

File: GI05MOAS-GL365/GI05MOAS-GL365_R00004_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00004/merged/gi*.mrg,GI05MOAS-GL365-00-ENG000000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00004/merged/gi*.mrg,GI05MOAS-GL365-01-FLORDM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00004/merged/gi*.mrg,GI05MOAS-GL365-02-DOSTAM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL365/R00004/merged/gi*.mrg,GI05MOAS-GL365-04-CTDGVM000,recovered_host,Not Expected,Glider lost at sea
```
Comment (snwhite) [inline GI05MOAS-GL365/GI05MOAS-GL365_R00004_ingest.csv:5]: Looks good.

File: GI05MOAS-GL453/GI05MOAS-GL453_R00002_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL453/R00002/merged/gi*.mrg,GI05MOAS-GL453-00-ENG000000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL453/R00002/merged/gi*.mrg,GI05MOAS-GL453-01-FLORDM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL453/R00002/merged/gi*.mrg,GI05MOAS-GL453-02-DOSTAM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL453/R00002/merged/gi*.mrg,GI05MOAS-GL453-04-CTDGVM000,recovered_host,Not Expected,Glider lost at sea
```
Comment (snwhite) [inline GI05MOAS-GL453/GI05MOAS-GL453_R00002_ingest.csv:5]: Looks good.

File: GI05MOAS-GL469/GI05MOAS-GL469_R00003_ingest.csv [hunk 8]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/R00003/merged/gi*.mrg,GI05MOAS-GL469-00-ENG000000,recovered_host,Not Expected,Recovered data not available
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/R00003/merged/gi*.mrg,GI05MOAS-GL469-01-FLORDM000,recovered_host,Not Expected,Recovered data not available
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/R00003/merged/gi*.mrg,GI05MOAS-GL469-02-DOSTAM000,recovered_host,Not Expected,Recovered data not available
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/R00003/merged/gi*.mrg,GI05MOAS-GL469-04-CTDGVM000,recovered_host,Not Expected,Recovered data not available
```
Comment (snwhite) [inline GI05MOAS-GL469/GI05MOAS-GL469_R00003_ingest.csv:5]: Looks good.

File: GI05MOAS-GL484/GI05MOAS-GL484_R00002_ingest.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
...
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00002/merged-from-glider/gi_*.mrg,GI05MOAS-GL484-04-CTDGVM000,recovered_host,Not Expected,lost glider
\ No newline at end of file
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00002/merged/gi_*.mrg,GI05MOAS-GL484-00-ENG000000,recovered_host,Not Expected,No recovered data expected
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00002/merged/gi_*.mrg,GI05MOAS-GL484-01-FLORDM000,recovered_host,Not Expected,No recovered data expected
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00002/merged/gi_*.mrg,GI05MOAS-GL484-02-DOSTAM000,recovered_host,Not Expected,No recovered data expected
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00002/merged/gi_*.mrg,GI05MOAS-GL484-04-CTDGVM000,recovered_host,Not Expected,No recovered data expected
```
Comment (snwhite) [inline GI05MOAS-GL484/GI05MOAS-GL484_R00002_ingest.csv:5]: Looks good.

File: GI05MOAS-GL484/GI05MOAS-GL484_R00003_ingest.csv [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
-#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,,GI05MOAS-GL484-02-DOSTAM000,recovered_host,Not Expected,Glider lost
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,,GI05MOAS-GL484-04-CTDGVM000,recovered_host,Not Expected,Glider lost
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00003/merged/gi_*.mrg,GI05MOAS-GL484-00-ENG000000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00003/merged/gi_*.mrg,GI05MOAS-GL484-01-FLORDM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00003/merged/gi_*.mrg,GI05MOAS-GL484-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL484/R00003/merged/gi_*.mrg,GI05MOAS-GL484-04-CTDGVM000,recovered_host,Not Expected,Glider lost
```
Comment (snwhite) [inline GI05MOAS-GL484/GI05MOAS-GL484_R00003_ingest.csv:5]: Looks good.

File: GI05MOAS-GL493/GI05MOAS-GL493_R00002_ingest.csv [hunk 11]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL493/R00002/merged/gi_*.mrg,GI05MOAS-GL493-00-ENG000000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL493/R00002/merged/gi_*.mrg,GI05MOAS-GL493-01-FLORDM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL493/R00002/merged/gi_*.mrg,GI05MOAS-GL493-02-DOSTAM000,recovered_host,Not Expected,Glider lost
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL493/R00002/merged/gi_*.mrg,GI05MOAS-GL493-04-CTDGVM000,recovered_host,Not Expected,Glider lost
```
Comment (snwhite) [inline GI05MOAS-GL493/GI05MOAS-GL493_R00002_ingest.csv:5]: Looks good.

File: GI05MOAS-GL525/GI05MOAS-GL525_R00002_ingest.csv [hunk 12]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/R00002/merged/gi_*.mrg,GI05MOAS-GL525-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/R00002/merged/gi_*.mrg,GI05MOAS-GL525-01-FLORDM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/R00002/merged/gi_*.mrg,GI05MOAS-GL525-02-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/R00002/merged/gi_*.mrg,GI05MOAS-GL525-04-CTDGVM000,recovered_host,Available,
```
Comment (snwhite) [inline GI05MOAS-GL525/GI05MOAS-GL525_R00002_ingest.csv:5]: Looks good.

File: GI05MOAS-GL560/GI05MOAS-GL560_R00002_ingest.csv [hunk 13]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL560/R00002/merged/gi_*.mrg,GI05MOAS-GL560-00-ENG000000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL560/R00002/merged/gi_*.mrg,GI05MOAS-GL560-01-FLORDM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL560/R00002/merged/gi_*.mrg,GI05MOAS-GL560-02-DOSTAM000,recovered_host,Not Expected,Glider lost at sea
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-GL560/R00002/merged/gi_*.mrg,GI05MOAS-GL560-04-CTDGVM000,recovered_host,Not Expected,Glider lost at sea
```
Comment (snwhite) [inline GI05MOAS-GL560/GI05MOAS-GL560_R00002_ingest.csv:5]: Looks good.

File: GI05MOAS-PG515/GI05MOAS-PG515_R00003_ingest.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/R00003/merged/gi*.mrg,GI05MOAS-PG515-02-DOSTAM000,recovered_host,Not Expected,Glider leaked. No data expected
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/R00003/merged/gi*.mrg,GI05MOAS-PG515-03-FLORTM000,recovered_host,Not Expected,Glider leaked. No data expected
+#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/R00003/merged/gi*.mrg,GI05MOAS-PG515-04-FLORTO000,recovered_host,Not Expected,Glider leaked. No data expected
+#mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/R00003/merged/gi*.mrg,GI05MOAS-PG515-05-NUTNRM000,recovered_host,Not Expected,Glider leaked. No data expected
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/R00003/merged/gi*.mrg,GI05MOAS-PG515-06-PARADM000,recovered_host,Not Expected,Glider leaked. No data expected
```
Comment (snwhite) [inline GI05MOAS-PG515/GI05MOAS-PG515_R00003_ingest.csv:8]: Looks good.

File: GP05MOAS-GL363/GP05MOAS-GL363_D00004_ingest.csv [hunk 15]
```diff
@@ -1,5 +1,5 @@
...
-mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00004/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-04-CTDGVM000,telemetered,Available,
\ No newline at end of file
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00004/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-00-ENG000000,telemetered,Not Expected,No data available
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00004/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,telemetered,Not Expected,No data available
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00004/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-02-DOSTAM000,telemetered,Not Expected,No data available
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00004/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-04-CTDGVM000,telemetered,Not Expected,No data available
```
Comment (snwhite) [inline GP05MOAS-GL363/GP05MOAS-GL363_D00004_ingest.csv:5]: Looks good.

File: GP05MOAS-GL363/GP05MOAS-GL363_R00004_ingest.csv [hunk 16]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00004/merged/gp_*.mrg,GP05MOAS-GL363-00-ENG000000,recovered_host,Not Expected,No recovered data available
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00004/merged/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,recovered_host,Not Expected,No recovered data available
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00004/merged/gp_*.mrg,GP05MOAS-GL363-02-DOSTAM000,recovered_host,Not Expected,No recovered data available
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00004/merged/gp_*.mrg,GP05MOAS-GL363-04-CTDGVM000,recovered_host,Not Expected,No recovered data available
```
Comment (snwhite) [inline GP05MOAS-GL363/GP05MOAS-GL363_R00004_ingest.csv:5]: Looks good.

File: GA05MOAS-PG578/GA05MOAS-PG578_R00001_ingest.csv (hunk 3 above)
Comment (cdobs) [reply GA05MOAS-PG578/GA05MOAS-PG578_R00001_ingest.csv:8]: The glider NUTNR driver is here: https://github.com/oceanobservatories/mi-instrument/tree/master/mi/dataset/driver/nutnr_m/glider while the rest of the glider drivers are here: https://github.com/oceanobservatories/mi-instrument/tree/master/mi/dataset/driver/moas/gl

I'm not sure if there was a reason it was kept separate from the rest, but an argument could definitely be made that it should be moved.

**Pull request merged**